icon_validator/rules/plugin_validators/dockerfile_parent_validator.py

```diff
@@ -7,11 +7,12 @@ def validate(self, spec):
         spec_str = "".join(spec.raw_dockerfile())
 
         valid_images = [
-            "komand/go-plugin-2", "komand/python-2-plugin", "komand/python-3-plugin", "komand/python-pypy3-plugin",
```

Comment:
Want to point this out to everyone that any future bug fixes that still use python 2 base image will fail the validators.  I think this is for the best 👍